### PR TITLE
Shine animation bgColor

### PR DESCRIPTION
**Pull request by felipemillhouse**
When the backgroundColor screen isn't '#FFFFFF' the Shine Animation broken. I just created a new optional prop 'backgroundColor' to fix that.

### CODE REVIEW

**Comment by mfrachet** [comment]
Awesome! Thanks for your help!

Let me make some experiments with your approach. I'll be back soon 😊

File: src/animation/shine.js [hunk 1]
```diff
@@ -11,14 +11,13 @@ const styles = StyleSheet.create({
...
     height: '100%',
-    backgroundColor: '#ffffff',
     opacity: 0.4,
   },
 });
 /**
  * Create a repetitive Shine animation
  */
-const Shine = ({ children }) => {
+const Shine = ({ backgroundColor, children }) => {
```
Comment (mfrachet) [inline src/animation/shine.js]: Can you rename it `animationColor` ? It makes more sense to be there. WYT?

Comment (felipemillhouse) [reply src/animation/shine.js]: Yeah, it's make more sense. I agree 100%

**Comment by mfrachet** [comment]
I've played a little with the pull request. Since the animation of this library relies on a "hack", it's a bit weird in live.

What can be done: only making animation on "primitives", like `line` or `media` elements and make the animation pass through.

WYT?

animationcolor

**Comment by felipemillhouse** [comment]
Yeah, I know... This is the reason that I called backgroundColor before, because my idea is to fix animation when background user is not #FFFFFF, in your case the background is #FFFFFF so, the animation needs to be #FFFFFF too.
If you do a test before that commit, with gray background you'll see a white square on animation. So let me try fix that returning property name to backgroundColor, and then changing the backgroundcolor and animationcolor at same moment.

**Comment by felipemillhouse** [comment]
Try now Marvin, the property name is 'backgroundColor' but now it changes backgroundColor and shine color.
But I agree with that the best way would be making animation just on "primitives", but I can't find the solution on code.

**Comment by mfrachet** [comment]
I close this for now, I think that I have a plan for a new release to make it work properly 😊

**Pull request closed without merging**